libs/validation/src/lib/decorators/string/IsStringWithNonzeroLength.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { validateSync, ValidationError, ValidationOptions } from 'class-validator';
+import { IsStringWithNonzeroLength } from './IsStringWithNonzeroLength';
+
+type HasTestProperty = {
+    testProperty: unknown;
+};
+
+export const checkValidValues = (
```

Comment:
Add a comment about how this differs from `class-validators` approach, and how it is the same.